piIOComm.c

```diff
@@ -476,7 +476,7 @@ EGpioValue piIoComm_readSniff1A()
 
 EGpioValue piIoComm_readSniff1B()
 {
-	if (piDev_g.pibridge_supported && !piDev_g.only_left_pibridge) {
+	if (!piDev_g.only_left_pibridge) {
```

Comment:
Ah here we go, this ended up in the wrong commit. Move it to the commit which converts everything to the new flags.